src/renderer/components/atoms/checkbox/types.ts

```diff
@@ -0,0 +1,9 @@
+import type { ReactNode } from 'react';
+
+export interface CheckboxProps {
+  id?: string;
+  checked?: boolean;
+  onChange?: (checked: boolean) => void;
+  children?: ReactNode;
+  disabled?: boolean;
```

Comment:
I think the only one optional must be children, what do you think?

Reply:
I think at this level we should be flexible with the props that we are going to receive since we are not accurately mapping all the scenarios on which this component will be used. We do need to mark as mandatory all the fields that are really needed for the component internals to work, but apparently in this component that is not the case for all the props:

- I'm using `id` just to link the `label` to the `input` and add the ability to check/uncheck the box if the label is clicked. If it's not being passed, I'm generating one id randomly, I don't really want to always pass an id every time this is needed, probably just on forms or something like that.
- `checked` and `onChange` are just props used to control the component outside. If it's not needed to control it (which is not very likely but might be the case), the component won't break.
- `children` just sets the content of what's going to be on the side of the checkbox. Maybe we'll just want in some cases a single checkbox without anything else
- `disabled` is false by default, no need to make it mandatory

Reply:
ok, lets go flexible!